meine_stadt_transparent/settings/__init__.py

```diff
@@ -3,19 +3,21 @@
 import os
 import subprocess
 import warnings
+
 from importlib.util import find_spec
-from logging import Filter, LogRecord
 from subprocess import CalledProcessError
 from typing import Dict, Union, Optional
+from pathlib import Path
 
 import sentry_sdk
 from sentry_sdk import configure_scope
 from sentry_sdk.integrations.django import DjangoIntegration
 from sentry_sdk.integrations.logging import ignore_logger
 
-from meine_stadt_transparent.settings.env import *
-from meine_stadt_transparent.settings.nested import *
-from meine_stadt_transparent.settings.security import *
+from meine_stadt_transparent.settings.env import env, TESTING
+from meine_stadt_transparent.settings.nested import INSTALLED_APPS, MIDDLEWARE
```

Comment:
Agreed. I added them again. We only need the specific imports to avoid any linting errors due to undefined variables as the linter does not resolve the * import.